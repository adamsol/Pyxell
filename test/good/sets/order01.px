
print {5, 8, 0, 5, -1} + {15, 4} + {0, 7, 0}
