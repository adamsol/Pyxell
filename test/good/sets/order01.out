{5, 8, 0, -1, 15, 4, 7}
